vscode/settings.json

```diff
@@ -138,6 +138,7 @@
   "ocaml.server.extraEnv": {
     "OCAMLLSP_SEMANTIC_HIGHLIGHTING": "full"
   },
+  "ocaml.server.args": ["--fallback-read-dot-merlin"],
```

Comment:
thanks to https://discuss.ocaml.org/t/how-to-pass-fallback-read-dot-merlin-to-ocamllsp-in-vscode-ocaml-platform-extension/10152/5